app/src/main/java/com/infomaniak/drive/ui/menu/MenuFragment.kt

```diff
@@ -65,6 +65,7 @@ class MenuFragment : Fragment() {
 
             AccountUtils.getCurrentDrive()?.let { currentDrive ->
                 driveName.text = currentDrive.name
+                driveName.setOnClickListener { safeNavigate(R.id.switchDriveDialog) }
```

Comment:
This duplicates the clickListener.
Maybe try with a container for the 2 icons + the text ?
And try to center the text between the 2 icons, it's not centred atm.

Reply:
Done.